src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -58,7 +58,7 @@
       <td class="align-center">
         <div class="checkbox">
           <label>
-            <input type="checkbox" class="table-column-no-float" name="<%= Const.ParamsNames.SUBMISSION_REMIND_USERLIST %>" value="${userToRemindEmail}">
+            <input id="all-students-responded" type="checkbox" class="table-column-no-float" name="<%= Const.ParamsNames.SUBMISSION_REMIND_USERLIST %>" value="${userToRemindEmail}">
```

Comment:
And this one.

Reply:
Also, I think id-ing them as "all students.... " is not really appropriate, and using `id` is not a good idea in the first place since the `id` is not unique. Using a `class` here is probably a better choice.

Reply:
I see. Thanks. In this case, then should I change it to all respondents then?

Reply:
I think something along the line of "responded-student" or "student-responded" is good.